meilisearch-http/src/routes/document.rs

```diff
@@ -29,6 +30,19 @@ macro_rules! guard_content_type {
 }
 
 guard_content_type!(guard_json, "application/json");
+*/
+
+fn guard_json(head: &actix_web::dev::RequestHead) -> bool {
+            if let Some(content_type) = head.headers.get("Content-Type") {
+                content_type
+                    .to_str()
```

Comment:
This returns an option whether the bytes from the header could be converted to a str (if I'm not mistaken), so you are saying that is we cannot convert the bytes to str, we assume it's json